src/helpers/get-int-config.helper.test.ts

```diff
@@ -0,0 +1,47 @@
+import { InvalidConfigException } from '@ukef/config/invalid-config.exception';
+
+import { getIntConfig } from './get-int-config';
+
+describe('Get integer configuration value Helper', () => {
```

Comment:
`GetIntConfig helper` matches our existing patterns

Reply:
Thanks, changed in Estore and MDM .